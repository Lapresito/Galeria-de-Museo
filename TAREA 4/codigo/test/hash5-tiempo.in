tiempoHashVisitaDia 100000 1
Fin