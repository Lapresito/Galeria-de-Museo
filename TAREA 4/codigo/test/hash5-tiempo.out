Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Comienzo de test de tiempo para Hash Visita Dia.
Fin de test de tiempo para Hash Visita Dia.
2> Fin.
